server_admin/topics/authentication/flows.adoc

```diff
@@ -2,197 +2,228 @@
-  After pressing "Save", set its Requirement to _Required_.
-* Using the `Actions` menu on the right-hand side of the "Authentication" subflow, select "Add execution". Using the drop-down select
-  "Webauthn Passwordless Authenticator". After pressing "Save", set its Requirement to _Alternative_.
-* Using the `Actions` menu on the right-hand side of the "Authentication" subflow, select "Add flow". Choose an representative Alias, e.g. "Password with OTP".
-  After pressing "Save", set its Requirement to _Alternative_.
-* Using the `Actions` menu on the right-hand side of the "Password with OTP" subflow, select "Add execution". Using the drop-down select
-  "Password Form". After pressing "Save", set its Requirement to Required.
-* Using the `Actions` menu on the right-hand side of the "Password with OTP" subflow, select "Add execution". Using the drop-down select
-  "OTP Form". After pressing "Save", set its Requirement to Required.
-* In the "Bindings" menu, change the browser flow from "Browser" to "Browser Password-less"
+If the user has WebAuthn passwordless credentials recorded, they can use them to login directly. This is the password-less login. The user can also select *Password with OTP* because the `WebAuthn Passwordless` execution and the `Password with OTP` flow are set to *Alternative*. If they are set to *Required*, the user has to enter WebAuthn, password, and OTP.
 
-The final flow that is produced is the following:
+If the user selects the *Try another way* link with `WebAuthn passwordless` authentication, they can choose between `Password` and `Security Key` (WebAuthn passwordless). When selecting the password, the user will need to continue and login with the assigned OTP. If the user has no WebAuthn credentials, they must enter the password and then the OTP. If the user has no OTP credential, they will be asked to record one.
 
-.A password-less browser login
-image:images/Passwordless-browser-login.png[]
+[NOTE]
+====
+Since the WebAuthn Passwordless execution is set to *Alternative* rather than  *Required*, this flow will never ask the user to register a WebAuthn credential. For a user to have a Webauthn credential, an administrator must add a required action to the user. Do this by:
+
+. Enabling the *Webauthn Register Passwordless* required action in the realm (see the <<_webauthn,WebAuthn>> documentation).
+. Setting the required action using the *Credential Reset* part of a user's xref:ref-user-credentials_{context}[Credentials] management menu.
+
+Creating an advanced flow such as this can have side effects. For example, if you enable the ability to reset the password for users, this would be accessible from the password form. In the default `Reset Credentials` flow, users must enter their username. Since the user has already entered their username earlier in the `Browser Password-less` flow, this is unnecessary for {project_name} and sub-optimal for user experience. To correct this, you can:
```

Comment:
, this action...
Since the user has already entered "a" username...  (their is plural)
To correct this "problem"...